packages/rancher-logging/generated-changes/patch/values.yaml.patch

```diff
@@ -198,7 +189,7 @@
 +  env: []
 +  logLevel: {}
 +  metrics:
-+    serviceMonitor: false
++    serviceMonitoring: false
```

Comment:
the value `serviceMonitor` has to be maintained in favor of `serviceMonitoring`

Reply:
woops, typo during forward port